src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkTask.java

```diff
@@ -153,11 +165,7 @@ public static Client createKustoEngineClient(KustoSinkConfig config) {
                             props.setIngestionMapping(mappingRef, IngestionMapping.IngestionMappingKind.Avro);
                         } else if (format.equalsIgnoreCase(IngestionProperties.DATA_FORMAT.apacheavro.toString())){
                             props.setIngestionMapping(mappingRef, IngestionMapping.IngestionMappingKind.ApacheAvro);
-                        } else if (format.equalsIgnoreCase(IngestionProperties.DATA_FORMAT.parquet.toString())) {
```

Comment:
we need to update the docs regarding parquet and orc

Reply:
updated the readme.md and the connector.sink.properties file. Connector documentation is also updated